.github/workflows/npm-lib-release.yml

```diff
@@ -140,6 +140,7 @@ jobs:
         with:
           name: ${{ steps.build.outputs.artifact }}
           path: ${{ steps.build.outputs.artifact }}
+          if-no-files-found: error
```

Comment:
I'm not sure it'll do what you expect. This will make it fail if the source is not available for the upload, if I read the docs properly, this will not fail the pipeline if the upload fails.

Reply:
Doesn't a step failing also fail the pipeline?

Reply:
yes, but this parameter has no effect if the upload itself fails...

Reply:
Okay, this option is helpful for the use case we have on hand, but I'll do a few more things in the flow to make this work better.